ansible/roles/fanout/templates/arista_7260cx3_deploy.j2

```diff
@@ -0,0 +1,152 @@
+!
+no schedule tech-support
+!
+transceiver qsfp default-mode 4x10G
+!
+no lldp run
+!
+hostname {{ inventory_hostname }}
+!
+ntp source vrf management Management1
+{% for ntp_server in ntp_servers %}
+ntp server vrf management {{ ntp_server }}
+{% endfor %}
+!
+spanning-tree mode none
+no spanning-tree vlan {{ device_vlan_range | list | join(',') }}
+!
+!
+aaa authorization exec default local
+aaa root secret 0 {{ lab_admin_pass }}
+!
+username admin privilege 15 role network-admin secret 0 {{ lab_admin_pass }}
+username eapi privilege 15 secret 0 {{ lab_admin_pass }}
+!
+vlan {{ device_vlan_range | list | join(',') }}
+!
+vrf definition management
+   rd 1:1
+!
+{% for i in range(1,65) %}
+{% set intf =  'Ethernet' + i|string + '/1'  %}
+{% if intf in device_port_vlans and device_port_vlans[intf]['mode'] != "Trunk" %}
+{%     if device_conn[intf]['speed'] == "100000" %}
+   interface {{ intf }}
+   description {{ device_conn[intf]['peerdevice'] }}-{{ device_conn[intf]['peerport'] }}
+   switchport access vlan {{ device_port_vlans[intf]['vlanids'] }}
+   switchport mode dot1q-tunnel
+   spanning-tree portfast
+   speed forced 100gfull
+   error-correction encoding reed-solomon
+   no shutdown
+{%     elif device_conn[intf]['speed'] == "40000" %}
+   interface {{ intf }}
+   description {{ device_conn[intf]['peerdevice'] }}-{{ device_conn[intf]['peerport'] }}
+   switchport access vlan {{ device_port_vlans[intf]['vlanids'] }}
+   switchport mode dot1q-tunnel
+   spanning-tree portfast
+   speed forced 40gfull
+   no shutdown
+{%     elif device_conn[intf]['speed'] == "10000" %}
+{%          for sub in range (1,5) %}
+{%             set subintf = 'Ethernet' + i|string + '/' + sub|string %}
+   interface {{ subintf }}
+   description {{ device_conn[intf]['peerdevice'] }}-{{ device_conn[intf]['peerport'] }}
+   switchport access vlan {{ device_port_vlans[intf]['vlanids'] }}
+   switchport mode dot1q-tunnel
+   spanning-tree portfast
+   speed forced 10gfull
+   no shutdown
+!
+{% endfor %}   
+{%     elif device_conn[intf]['speed'] == "50000" %}
+interface {{ intf }}
+   description {{ device_conn[intf]['peerdevice'] }}-{{ device_conn[intf]['peerport'] }}
+   switchport access vlan {{ device_port_vlans[intf]['vlanids'] }}
+   switchport mode dot1q-tunnel
+   spanning-tree portfast
+   speed forced 50gfull
+   no error-correction encoding
+   no shutdown
+!
+{% set intf =  'Ethernet' + i|string + '/3'  %}
+interface {{ intf }}
+{%         if intf in device_port_vlans and device_port_vlans[intf]['mode'] != "Trunk" %}
+   description {{ device_conn[intf]['peerdevice'] }}-{{ device_conn[intf]['peerport'] }}
+   switchport access vlan {{ device_port_vlans[intf]['vlanids'] }}
+   switchport mode dot1q-tunnel
+   spanning-tree portfast
+   speed forced 50gfull
+   no error-correction encoding
+   no shutdown
+{%         else %}
+   shutdown
+{%         endif %}
+{%     else %}
+{#        Not valid speed value #}
+   shutdown
+{%     endif %}
```

Comment:
add ! here. we might not configure this intf properly if this section of code is hit. '!' is used to separate out each config chunk. if its missed, then the config will get overwritten with the next chunk

Reply:
I tried to read this conditionals carefully. It seems like this `else` part's `if` statement already has a `!` before it..
```
!
if {}
elif {}
else {}
!
```

Can you double check if it is still needed?

Reply:
I see that covered in 108. All good